Tests/Test_BSP_Contactors.c

```diff
@@ -0,0 +1,12 @@
+#include "common.h"
```

Comment:
After making the changes to this file, leave documentation on how this test should be used.

Reply:
Documented what is being tested and how it should be used.